mne/forward/forward.py

```diff
@@ -875,6 +889,33 @@ def write_forward_meas_info(fid, info):
     end_block(fid, FIFF.FIFFB_MNE_PARENT_MEAS_FILE)
 
 
+def _check_loose(forward, loose):
+    """Check loose parameter input."""
+    if loose is not None:
+        if is_fixed_orient(forward):
+            warn('Ignoring loose parameter with forward operator '
+                 'with fixed orientation.')
+            loose = None
+        if isinstance(loose, float):
+            if not (0 <= loose <= 1):
+                raise ValueError('Loose value should be smaller than 1 and '
+                                 'bigger than 0, or None for not loose '
+                                 'orientations.')
+            if loose < 1 and not forward['surf_ori']:
+                warn('Forward operator is not oriented in surface '
+                     'coordinates. A loose inverse operator requires a '
+                     'surface-oriented forward operator with free '
+                     'orientation.')
+                forward = convert_forward_solution(forward, surf_ori=True)
+        else:
+            raise ValueError('loose value must be None for not loose '
```

Comment:
loose must be None for not loose orientations ?

this is unclear what you mean here.